src/test/java/seedu/address/logic/commands/ExpenseCommandTest.java

```diff
@@ -32,37 +31,18 @@
     public static void setLists() {
         persons.add("Pauline");
         persons.add("Benson");
-        amounts.add(new Amount(10));
-        amounts.add(new Amount(20));
-        expenses.add(new Expense(TypicalPersons.ALICE.getPrimaryKey(), amounts.get(0), emptyString));
-        expenses.add(new Expense(TypicalPersons.BENSON.getPrimaryKey(), amounts.get(1), emptyString));
-        command = new ExpenseCommand(persons, amounts, emptyString);
+        expenses.add(new Expense(TypicalPersons.ALICE.getPrimaryKey(), amount, emptyString));
+        command = new ExpenseCommand(persons, amount, emptyString);
     }
 
     @Test
     public void constructor_nullParams_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () ->
-                new ExpenseCommand(null, new ArrayList<>(), emptyString));
+                new ExpenseCommand(null, new Amount(1), emptyString));
```

Comment:
Assuming we go through with the idea that we shouldn't allow `null` to be passed into the expense constructor that takes in ids then should add one more line to test

Reply:
This is for `ExpenseCommand`. The constructor does not support the fourth param.